pkg/client/cli/intercept/command.go

```diff
@@ -105,6 +110,20 @@ func (c *Command) AddInterceptFlags(cmd *cobra.Command) {
 		flagSet.Lookup("replace").Deprecated = "Use the replace command."
 	}
 
+	// HTTP Intercepts flags
+	flagSet.StringSliceVar(&c.HeaderFilters, "header", nil,
+		`HTTP header filters for HTTP Intercepts. Only requests with matching headers will be intercepted. `+
+			`Format: --header "X-User-ID=dev123" --header "X-Environment=staging". Multiple headers use AND logic. `+
+			`Requires --http flag.`)
+
+	flagSet.StringSliceVar(&c.PathFilters, "path", nil,
```

Comment:
Would be nice if this was extended to equal, prefix, and regexp matches, just like the pro-version has.